wallaby.h

```diff
@@ -17,6 +17,11 @@
 #include "wallaby_uart.h"
 
 
+// Wallaby2
+#ifndef WALLABY2
+#define WALLABY2

```

Comment:
Don't default to ``WALLABY2`` in this way if it is missing... what if we were a ``WALLABY``